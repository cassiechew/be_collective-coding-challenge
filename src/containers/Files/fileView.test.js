import { create } from 'react-test-renderer';
import React from 'react';
import FileView from './fileView';

describe('snapshot test', () => {
    test('testing footer', () => {
      let tree = create(<FileView setParentNumFiles={null} setParentFileSize={null} />);
      expect(tree.toJSON()).toMatchSnapshot();
    });
});
