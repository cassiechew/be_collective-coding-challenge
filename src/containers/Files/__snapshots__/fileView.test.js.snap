// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test testing footer 1`] = `
<div
  className="container"
>
  <div
    className="container__file-list"
  />
</div>
`;
